src/lib/components/multiDropzone/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useCallback } from 'react';
+import { useDropzone } from 'react-dropzone';
+import classnames from 'classnames';
+
+import styles from './style.module.scss';
+import icons from './icons/index';
+import UploadFileCell from './UploadFileCell';
+
+export type UploadStatus = 'UPLOADING' | 'COMPLETE' | 'ERROR';
+
+const getUploadStatus = (progress: number, error?: string): UploadStatus => {
+  if (error) {
+    return 'ERROR';
+  }
+
+  if (progress < 100) {
+    return 'UPLOADING';
+  }
+
+  return 'COMPLETE';
+};
+
+export interface UploadedFile {
+  id: string;
+  name: string;
+  type: string;
+  previewUrl?: string;
+  progress: number;
+  error?: string;
+}
+
+interface Props {
+  onFileSelect: (file: File) => void;
+  uploadedFiles: UploadedFile[];
+  disabled: boolean;
+  onRemoveFile: (id: string) => void;
+}
+
+export default ({
+  uploadedFiles,
+  onFileSelect,
+  disabled,
+  onRemoveFile,
+}: Props) => {
+  const onDrop = useCallback(
+    acceptedFiles => {
+      onFileSelect(acceptedFiles[0]);
+    },
+    [onFileSelect]
+  );
+
+  const { getRootProps, getInputProps } = useDropzone({ onDrop });
+
+  return (
+    <div className={`w100 ${styles.container}`}>
+      <div
+        className={classnames(`w100 ta-center ${styles['upload-container']}`, {
+          [styles['upload-container-disabled']]: disabled,
+        })}
+        {...getRootProps()}
+      >
+        <input {...getInputProps()} />
+        <img src={icons.uploadIcon} alt="" />
+        <div className="p-h4 mt8">
+          {disabled
+            ? 'Please wait while uploading file...'
+            : 'Choose file or drag & drop'}
+        </div>
```

Comment:
`disabled` doesn't seem to be a good name here, as far as I can see when disabled the text "Please wait while uploading file..." is being displayed. Maybe a better wording would be `uploading`?